### PR TITLE
Add link to service in CloudHost change view

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

File: src/ralph/virtual/admin.py [hunk 1]
```diff
@@ -310,6 +310,19 @@ def get_cloudproject(self, obj):
     get_cloudproject.allow_tags = True
     get_cloudproject._permission_field = 'parent'
 
+    def get_service(self, obj):
+        return '<a href="{}">{}</a>'.format(
```
Comment (mkurek) [inline src/ralph/virtual/admin.py]: what if obj has no service_env?

**Review by mkurek — CHANGES_REQUESTED** [review]

File: src/ralph/virtual/admin.py (hunk 1 above)
Comment (szok) [reply src/ralph/virtual/admin.py]: Done

File: src/ralph/virtual/admin.py [hunk 2]
```diff
@@ -310,6 +310,21 @@ def get_cloudproject(self, obj):
     get_cloudproject.allow_tags = True
     get_cloudproject._permission_field = 'parent'
 
+    def get_service(self, obj):
+        if self.service_env_id:
+            return '<a href="{}">{}</a>'.format(
+                reverse(
+                    "admin:assets_service_change",
+                    args=(obj.service_env.service.id,)
```
Comment (mkurek) [inline src/ralph/virtual/admin.py]: obj.service_env.service.id -> obj.service_env.service_id

File: src/ralph/virtual/admin.py [hunk 3]
```diff
@@ -310,6 +310,21 @@ def get_cloudproject(self, obj):
...
+                ),
+                obj.service_env
+            )
+        return ''
+    get_service.short_description = _('Service env')
+    get_service.admin_order_field = 'parent'
```
Comment (mkurek) [inline src/ralph/virtual/admin.py]: `parent`? why not `service_env`

File: src/ralph/virtual/admin.py [hunk 4]
```diff
@@ -310,6 +310,21 @@ def get_cloudproject(self, obj):
...
+    get_service.allow_tags = True
+    get_service._permission_field = 'parent'
```
Comment (mkurek) [inline src/ralph/virtual/admin.py]: ditto

**Review by mkurek — CHANGES_REQUESTED** [review]

File: src/ralph/virtual/admin.py (hunk 3 above)
Comment (szok) [reply src/ralph/virtual/admin.py]: done

File: src/ralph/virtual/admin.py (hunk 4 above)
Comment (szok) [reply src/ralph/virtual/admin.py]: done

File: src/ralph/virtual/admin.py (hunk 2 above)
Comment (szok) [reply src/ralph/virtual/admin.py]: done, but it is not necessary, because I have get str(obj.service_env.service) in line 320

**Review by mkurek — APPROVED** [review]

**Pull request merged**